core/domain/platform_parameter_list.py

```diff
@@ -54,6 +54,7 @@ class ParamName(enum.Enum):
     UNPUBLISH_EXPLORATION_EMAIL_HTML_BODY = (
         'unpublish_exploration_email_html_body')
     RECORD_PLAYTHROUGH_PROBABILITY = 'record_playthrough_probability'
+    ENABLE_VOICEOVER_CONTRIBUTION = 'enable_voiceover_contribution'
```

Comment:
Do you need this & the feature flag? Generally just the feature flag is sufficient. These values are used for configuration values.

Or, if this is needed, how will this value be used vs. the feature flag?

Reply:
Removed, thank you for pointing this.